src/Dropdown/Dropdown.js

```diff
@@ -1,10 +1,15 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
-import withStyles from '../utils/WithStyles/WithStyles';
+import React, { useEffect } from 'react';
 
 const Dropdown = React.forwardRef(({ children, className, disableStyles, standard, ...props }, ref) => {
 
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/dropdown.css');
```

Comment:
should I see `dropdown.css` in fundamental-styles?  https://github.com/SAP/fundamental-styles/tree/master/src ?  Because I don't

Reply:
looks like it exists in 0.4.0 though, nevermind